src/Components/Admin/admin.js

```diff
@@ -245,6 +254,60 @@ export default function Admin() {
         setDeleteModalOpen(false);
     };
 
+    const handleUploadAssetClose = () => {
+        setUploadAssetModalOpen(false);
+    };
+
+    const handleUploadAssetSubmit = async (
```

Comment:
@vivianliu0 is it possible to turn this into a utility function and put it in a common file? Specifically, the first part of it, where it takes in a file_type, type, s3_key, the file details and uploads to s3 (until line 296).  Since this will be something that will be used throughout (like for image uploads)

Reply:
Lol I literally said the same thing at the same time

Reply:
Thanks guys, updated PR and extracted the s3 utility functions into a separate file